blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/UserImpl.java

```diff
@@ -155,41 +155,41 @@ public Link getLink() {
      * Gets or creates the user's private Jenkins-managed key and returns the
      * public key to the user
      * @return JSON response
-     * @throws IOException 
+     * @throws IOException
      */
     @GET
     @WebMethod(name="publickey")
-    public HttpResponse publicKey() throws IOException {
+    public HttpResponse publicKey() {
         User authenticatedUser =  User.current();
         if (authenticatedUser == null) {
             throw new ServiceException.UnauthorizedException("Not authorized");
         }
         if (!StringUtils.equals(getId(), authenticatedUser.getId())) {
             throw new ServiceException.ForbiddenException("Not authorized");
         }
-        
-        String publicKey = UserSSHKeyManager.getReadablePublicKey(authenticatedUser, 
+
+        String publicKey = UserSSHKeyManager.getPublicKey(authenticatedUser,
             UserSSHKeyManager.getOrCreate(authenticatedUser));
-        
+
         return HttpResponses.okJSON(ImmutableMap.of("key", publicKey));
```

Comment:
Could this return its own model like other methods in our API?
``
@ExportedBean
public class UserKey {
  private final publicKey;

  @Exported(name="publickey")
  public String getPublicKey() {
    return publicKey;
  }
}

```